packages/website/docusaurus.config.js

```diff
@@ -51,6 +52,7 @@ const config = {
       }),
     ],
   ],
+  staticDirectories: ["../../docs"],
```

Comment:
This is a bit of a workaround as it will make everything in `/docs` a static file. There is some leftover docs in there for now that we should probably remove but  his opened a few questions:
1. Do we need `spec.html` committed in the repo?
2. If yes should it stay under `docs`? Would `docs/spec/spec.html` be good
3. We can actually with docusaurus set the root of the docs to be outside the website package. Which means we can move the docs back up to the root of the repo. Should we do that?

Reply:
Re 1 & 2, I don't think we need spec.html committed in the repo. In fact, I think it would be great if we could move it to a tmp folder now that we can finally publish it.

Re 3. It would be nice to move the docs up if that works, but I'd do that separately unless doing it now will help get this in.

Reply:
ok will uncommit spec.html and have it generated as part of the build and added to the website that way.

For 3. will make another pr.